CHANGELOG.md

```diff
@@ -4,7 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.5.2] - 2023-02-07
```

Comment:
```suggestion
## [Unreleased]
```

We can just leave it as unreleased while we are developing. I'll rename the section and add the date when I create a new release.